besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1408,6 +1408,10 @@ public void run() {
         if (genesisConfigOptions.isQuorum()) {
           enableGoQuorumCompatibilityMode();
         }
+
+        if (genesisConfigOptions.isZeroBaseFee()) {
```

Comment:
Is this truly needed?

Reply:
Not really, it's more of a convenience. I'm happy to remove.